pubspec.yaml

```diff
@@ -33,10 +33,10 @@ dependencies:
     sdk: flutter
 
 dev_dependencies:
-  ffigen: ^2.3.0
+  ffigen: ^6.0.0
```

Comment:
[ffigen](https://pub.dev/packages/ffigen)

Reply:
We probably have to consider regenerating, if we're upgrading ffigen... but I can do that separately...